rubygem-smart_proxy_remote_execution_ssh/rubygem-smart_proxy_remote_execution_ssh.spec

```diff
@@ -35,6 +32,7 @@ BuildRequires: ruby(release)
 %endif
 BuildRequires: rubygems-devel
 BuildArch: noarch
+Obsoletes: %{?rhel:tfm-}rubygem(smart_proxy_remote_execution_ssh_core) < 0.1.4
```

Comment:
This doesn't seem to work for me, the original package remains installed:

<pre>
# installed the old 0.1.3 version:
[..]
  Installing : rubygem-smart_proxy_remote_execution_ssh-0.1.3-1.el7.noarch                                                                                                                                                                                                  1/1
  Verifying  : rubygem-smart_proxy_remote_execution_ssh-0.1.3-1.el7.noarch                                                                                                                                                                                                  1/1

Installed:
  rubygem-smart_proxy_remote_execution_ssh.noarch 0:0.1.3-1.el7

Complete!
# yum upgrade "http://koji.katello.org/koji/getfile?taskID=532542&name=rubygem-smart_proxy_remote_execution_ssh-0.1.4-1.git.109.33db759.el7.noarch.rpm"
Loaded plugins: fastestmirror
getfile?taskID=532542&name=rubygem-smart_proxy_remote_execution_ssh-0.1.4-1.git.109.33db759.el7.noarch.rpm                                                                                                                                               |  20 kB  00:00:00
Examining /var/tmp/yum-root-6kJJnP/getfile?taskID=532542&name=rubygem-smart_proxy_remote_execution_ssh-0.1.4-1.git.109.33db759.el7.noarch.rpm: rubygem-smart_proxy_remote_execution_ssh-0.1.4-1.git.109.33db759.el7.noarch
Marking /var/tmp/yum-root-6kJJnP/getfile?taskID=532542&name=rubygem-smart_proxy_remote_execution_ssh-0.1.4-1.git.109.33db759.el7.noarch.rpm as an update to rubygem-smart_proxy_remote_execution_ssh-0.1.3-1.el7.noarch
Resolving Dependencies
--> Running transaction check
---> Package rubygem-smart_proxy_remote_execution_ssh.noarch 0:0.1.3-1.el7 will be updated
---> Package rubygem-smart_proxy_remote_execution_ssh.noarch 0:0.1.4-1.git.109.33db759.el7 will be an update
--> Finished Dependency Resolution

Dependencies Resolved

================================================================================================================================================================================================================================================================================
 Package                                                       Arch                        Version                                           Repository                                                                                                                    Size
================================================================================================================================================================================================================================================================================
Updating:
 rubygem-smart_proxy_remote_execution_ssh                      noarch                      0.1.4-1.git.109.33db759.el7                       /getfile?taskID=532542&name=rubygem-smart_proxy_remote_execution_ssh-0.1.4-1.git.109.33db759.el7.noarch                       39 k

Transaction Summary
================================================================================================================================================================================================================================================================================
Upgrade  1 Package

Total size: 39 k
Is this ok [y/d/N]: y
Downloading packages:
Running transaction check
Running transaction test
Transaction test succeeded
Running transaction
  Updating   : rubygem-smart_proxy_remote_execution_ssh-0.1.4-1.git.109.33db759.el7.noarch                                                                                                                                                                                  1/2
  Cleanup    : rubygem-smart_proxy_remote_execution_ssh-0.1.3-1.el7.noarch                                                                                                                                                                                                  2/2
  Verifying  : rubygem-smart_proxy_remote_execution_ssh-0.1.4-1.git.109.33db759.el7.noarch                                                                                                                                                                                  1/2
  Verifying  : rubygem-smart_proxy_remote_execution_ssh-0.1.3-1.el7.noarch                                                                                                                                                                                                  2/2

Updated:
  rubygem-smart_proxy_remote_execution_ssh.noarch 0:0.1.4-1.git.109.33db759.el7

Complete!
# rpm -q --obsoletes rubygem-smart_proxy_remote_execution_ssh
tfm-rubygem(smart_proxy_remote_execution_ssh_core) < 0.1.4
# rpm -q --whatprovides "tfm-rubygem(smart_proxy_remote_execution_ssh_core)"
tfm-rubygem-smart_proxy_remote_execution_ssh_core-0.1.3-1.fm1_13.el7.noarch
</pre>

My only guess is that this must be the full package name, not the rubygem(..) provide, so try replacing `()` with the hyphenated package name and see if it works then.

Also, please add a similar line to the -doc subpackage with the -doc suffix, see some of the ruby193 obsoletes for examples.